### DISCUSSION TITLE
Suggestion: let user choose the way of creating tag.

### DISCUSSION THREAD

**Discussion by ghost** [foambubble/foam#1237] [General]
Hi. 

I installed foam today and love most of the new features, except the `#` tag creation. My work needs me to use `#` very often, and leaving me a lot of unnecessary tags. 

So I would be very happy and thankful if I can disable the `#` way and create tags only by `tags: `.

Thanks for your work!

**Comment by riccardoferretti** [comment]
I could add the extra configuration into Foam if useful for many (the tradeoff being more complexity), so I am happy to leave this discussion open to see if other people have the similar needs
